includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -715,6 +719,16 @@ public function list_disputes() {
 		return $disputes;
 	}
 
+	/**
+	 * Get summary of disputes.
+	 *
+	 * @return array
+	 * @throws API_Exception - Exception thrown on request failure.
+	 */
+	public function get_disputes_summary() {
```

Comment:
I believe we can make the parameters/return type strictly typed here.

Reply:
Done thanks